File: sdkwork-ai-sdk-csharp/Models/ModerationResponse.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class ModerationResponse
    {
        public string? Id { get; set; }
        public string? Model { get; set; }
        public List<ModerationResult>? Results { get; set; }
    }
}

File: sdkwork-ai-sdk-typescript/src/types/moderation-response.ts
import type { ModerationResult } from './moderation-result';

/** OpenAI标准审核响应 */
export interface ModerationResponse {
  /** The unique identifier for the moderation request */
  id?: string;
  /** The model used to generate the moderation results */
  model?: string;
  /** A list of moderation objects */
  results?: ModerationResult[];
}
